devdocs/staticAnalysis.md

```diff
@@ -0,0 +1,107 @@
+
+# Static Analysis
+
+TEAMMATES uses a number of static analysis tools in order to maintain code quality and measure code coverage.
+This document will cover an overview of these tools and how to run them in local environment.
+
+- [Tool stack](#tool-stack)
+- [Running static analysis](#running-static-analysis)
+- [Running code coverage session](#running-code-coverage-session)
+
+## Tool stack
```

Comment:
We already have another `Tool stack` section in another place, and it contains all other tools used. Add this part to that document too?.